ui/src/views/Settings.vue

```diff
@@ -241,12 +290,14 @@ export default {
       is_default_password_admin: false,
       is_default_password_auditor: false,
       isLetsEncryptEnabled: false,
+      letsEncryptIsEnabled: false,
```

Comment:
`letsEncryptIsEnabled` is too similar to `isLetsEncryptEnabled`, I suggest to rename this variable to something like `isLetsEncryptCurrentlyEnabled`